Documentation/upgrade.md

```diff
@@ -199,70 +173,39 @@ kubectl -n rook exec -it rook-tools -- ceph status
-we will need to edit the image version of the pod template in the deployment spec as well as add two environment variables.
-Begin editing the deployment with:
+The Rook API service has been removed. Delete the service and its deployment with the following commands:
 ```bash
-kubectl -n rook edit deployment rook-api
+kubectl -n rook delete svc rook-api
+kubectl -n rook delete deploy rook-api
 ```
 
-First update the container image version
-```
-        image: rook/rook:v0.7.0
-```
-
-Now add the two new environment variables, ensuring that the spacing is consistent with other variables.
-```bash
-        - name: POD_NAME
-          valueFrom:
-            fieldRef:
-              apiVersion: v1
-              fieldPath: metadata.name
-        - name: POD_NAMESPACE
-          valueFrom:
-            fieldRef:
-              apiVersion: v1
-              fieldPath: metadata.namespace
-```
-
-After updating the deployment, the old `rook-api` pod should terminate a new one will start using the new version.
-We can verify the new pod is in the `Running` state and using the new version with these commands:
-```bash
-API_POD_NAME=$(kubectl -n rook get pod -l app=rook-api -o jsonpath='{.items[0].metadata.name}')
-kubectl -n rook get pod ${API_POD_NAME} -o jsonpath='{.status.phase}{"\n"}{.spec.containers[0].image}{"\n"}'
-```
-
-Remember to verify the cluster health using the instructions found in the [health verification section](#health-verification).
-
 ### Monitors
 There are multiple monitor pods to upgrade and they are each individually managed by their own replica set.
-**For each** monitor's replica set, you will need to update the pod template spec's image version field to `rook/rook:v0.7.0`.
+**For each** monitor's replica set, you will need to update the pod template spec's image version field to `rook/rook:v0.7.1-4.g4233e03`.
 For example, we can update the replica set for `mon0` with:
 ```bash
-kubectl -n rook set image replicaset/rook-ceph-mon0 rook-ceph-mon=rook/rook:v0.7.0
+kubectl -n rook set image replicaset/rook-ceph-mon0 rook-ceph-mon=rook/rook:v0.7.1-4.g4233e03
 ```
 
 Once the replica set has been updated, we need to manually terminate the old pod which will trigger the replica set to create a new pod using the new version.
 ```bash
-MON0_POD_NAME=$(kubectl -n rook get pod -l mon=rook-ceph-mon0 -o jsonpath='{.items[0].metadata.name}')
-kubectl -n rook delete pod ${MON0_POD_NAME}
+kubectl -n rook delete pod -l mon=rook-ceph-mon0
 ```
 
 After the new monitor pod comes up, we can verify that it's in the `Running` state and on the new version:
 ```bash
-MON0_POD_NAME=$(kubectl -n rook get pod -l mon=rook-ceph-mon0 -o jsonpath='{.items[0].metadata.name}')
-kubectl -n rook get pod ${MON0_POD_NAME} -o jsonpath='{.status.phase}{"\n"}{.spec.containers[0].image}{"\n"}'
+kubectl -n rook get pod -l mon=rook-ceph-mon0 -o jsonpath='{.items[0].status.phase}{"\n"}{.items[0].spec.containers[0].image}{"\n"}'
```

Comment:
is this addition of a newline important?

Reply:
the newline looks the same to me. The change here was to use the label so you don't need a separate command to get the pod name